client/src/components/App.jsx

```diff
@@ -55,6 +60,12 @@ class App extends React.Component {
         <h2>
           <p> Price: ${this.state.displayProduct.default_price}</p>
         </h2>
+        <br></br>
+        <h3>Related products:</h3><br></br>
```

Comment:
*just a thought for you Andrew*

I notice you are using the break tags (br) - you can continue doing that or you can simply edit the spacing using CSS. I try to use CSS where i can instead of (br) to keep my HTML looking clean but not sure if it really matters :)

Reply:
great Thank you Aaron I added related-prod class to App.css ;